cmd/google_cloud_ops_agent_diagnostics/main_others.go

```diff
@@ -0,0 +1,61 @@
+// Copyright 2020 Google LLC
```

Comment:
We conventionally refer to "non-Windows" as just "Linux", so maybe we can call this file `main_linux.go`?

Reply:
It makes sense to call it `main_linux.go` since the implementation is `linux` specific (e.g. the signals used I think only come up in linux). Will update the name.